### PR TITLE
OSDOCS-9706: Deprecation status of SDN for 4.15 release notes

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9706

Link to docs previews:

(deprecation statement)  https://71848--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-deprecation-sdn

(deprecation table) https://71848--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-deprecated-removed-features

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@weliang1 Can you please provide QE review? Thank you!

**Comment by sdodson** [comment]
/lgtm
Thanks

**Comment by ahardin-rh** [comment]
Approval received by Marc Curry, PM, in Slack

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1146,6 +1151,11 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-15-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-15-deprecation-sdn"]
+==== Deprecation of the OpenShift SDN network plugin
+
+OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} {product-version}, the network plugin is not an option for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```
Comment (skopacz1) [inline release_notes/ocp-4-15-release-notes.adoc:1157]: Small suggestion, up to you:
```suggestion
OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} {product-version}, the network plugin is not available for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```
I'm not sure that would even be accurate, so feel free to ignore.

Comment (skopacz1) [inline release_notes/ocp-4-15-release-notes.adoc:1157]: Small suggestion again:
```suggestion
OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} {product-version}, the network plugin is not an option for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until it is removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```

**Review by skopacz1 — COMMENTED** [review]
Left a few suggestions but otherwise looks good to me!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-15-release-notes.adoc:1157]: Thank you! I think "not an option" is most accurate and that language is approved by PM. I will leave this part as-is, but appreciate it!

**Comment by ahardin-rh** [comment]
@jhou1 @weliang1 Can you please provide QE review? Thank you!

**Comment by stevsmit** [comment]
> @ahardin-rh Just wondering if we need to mention that we support SDN live migration to OVN through #71390.
> 
> Otherwise /LGTM

I'm hesitant to announce this publicly because the docs are only going to be circulated internally for now? If we do call it out I suppose we'd want to say something like "Contact your support engineer for a live migration procedure", or contact whoever is going to be in charge of executing that process on behalf of customers.

**Pull request merged**